### ISSUE TITLE
genes.jar not found

### ISSUE THREAD

**Issue by ngiangre** [ncbi/ngs-tools#4]
Hi, I followed the installation from the closed issue, and I was able to configure ngs/ ngs-tools/ and ncbi-vdb/ and sra-tools/. However, in the README for spark_genes it says I should have a genes.jar file along with other installations. If I can't find genes.jar does that mean I compiled wrong last time and is there installation instructions for the 3 dependencies listed? Thanks!

**Comment by kwrodarmer** [comment]
Can you point me to spark_genes and their README?

**Comment by ngiangre** [comment]
ngs-tools/tools/spark_genes/README.txt

**Comment by kwrodarmer** [comment]
Sorry - I thought you were referring to some other project with that name! Oops.

The `genes.jar` is created by the build. It is the output of the `spark_genes` script `build.sh`.

**Comment by ngiangre** [comment]
Ok-so I have to execute build.sh in the directory as well, correct? That wasn't laid out in the docs so I didn't know I had to do that.

**Comment by kwrodarmer** [comment]
Yes, sorry for the confusion - this is not part of our actual release.

We have code in our repositories that does not get built. If we are releasing it and standing behind it, it gets built and installed. For some of the side or experimental projects, the code is there but does not get built by the main `make`. This is one of the reasons you caught me by surprise.

`spark_genes` does work, but you may have to poke around a bit to figure out how, or wait until Monday when its author is back in the office to comment.

**Comment by ngiangre** [comment]
Ok, sounds good, thanks!

I spoke with Ben Busby about a problem I was facing (I need count data from a bunch of NGS data) and he said this would be a great tool to use instead of downloading bam files so I'm trying to see how I can use the tool.

If there's more stable and documented versions let me know!

**Comment by kwrodarmer** [comment]
Thanks, will do. The same author just wrote a tool that performs the basic work necessary for variant calling directly from SRA, but it is still in (working) prototype shape and we have more work to do on it. It's pretty cool!

**Comment by ngiangre** [comment]
Yeah it all seems really useful, thanks for the help! I'll wait until the author can get back to me.
